README.md

```diff
@@ -58,32 +123,300 @@ client = Redis.new(semian: {
+with single threaded web servers that are expected to return fast such as
+Unicorn this is problematic. While Semian is only known to have been used in
+production for web servers, there's nothing preventing it from being used in
+another kind of deployment.
+
+### Real World Example
+
+This is better illustrated with a real world example from Shopify. If you are
+browsing a store and you are signed in, Shopify stores your session in a session
+store. If the session store becomes unavailable, the data driver will start
+throwing driver errors. We rescue these exceptions and simply disable all
+customer sign in functionality on the store until the session store is back
+online.
+
+This is great if querying the resource fails instantly, because it means we fail
+in just a single inter-dc roundtrip of `~1ms`. But if the resource is
+unresponsive, or slow, this can take as long as our timeout which is easily
+`200ms`. This means every request, even if it does rescue the exception, now
+takes an extra `200ms`. Because every resource takes that long, our capacity is
+also significantly degraded. These problems are explained in depth in the next
+two sections.
+
+With Semian, the slow resource would fail instantly (after a small amount of
+convergence time) preventing your response time from spiking and not decreasing
+capacity of the cluster.
+
+If this sounds familiar to you, Semian is what you need to be resilient to
+latency. You may not need the graceful fallback depending on your application,
+in which case it will just result in an error (e.g. a `500`) quicker which is
+usually a good thing.
+
+We will now examine the two problems in more details.
+
+#### In depth analysis of real world example
+
+If a single resource is slow every single request is going to suffer. We saw
+this in the example before. Let's illustrate this more clearly in the following
+Rails example where the user session is stored in Redis:
+
+```ruby
+def index
+  @user = fetch_user
+  @posts = Post.all
+end
+
+private
+def fetch_user
+  user = User.find(session[:user_id])
+rescue Redis::CannotConnectError
+  nil
+end
+```
+
+Our code is resilient to a failure of the session layer, it doesn't `HTTP 500`
+if the session store is unavailable (this can be tested with
+[Toxiproxy][toxiproxy]). It'll still send a `500` back if the data store where
+users and posts are stored is unavailable, but we accept that (this could be
+prevented with a caching tier or something else out of scope). 
+
+However, this code has two flaws:
+
+1. **What happens if the session storage is consistently slow?** I.e. the majority
+   of requests take, say, more than half the timeout time (but it should only
+   take `~1ms`)?
+2. **What happens if the session storage is unavailable and is not responding at
+   all?** I.e. we hit timeouts on every request.
+
+These two problems in turn have two related problems associated with them:
+request time and capacity.
+
+#### Response time
```

Comment:
You are using "response time" and "request time" interchangeably... Can we pick one and use it consistently? (I would say you mean "response time")